app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Same here and as the last reviewer mentioned, please edit this line and create comments migration to make the foreign key `author_id` instead of `user_id`